content/en/blog/_posts/2023-03-06-node-log-viewer-alpha.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: blog
+title: "Node Log Viewer"
```

Comment:
```suggestion
title: "Kubernetes 1.27: Access Node Logs Using The Kubernetes API"
```
“viewer” implies there's a client tool available.

Reply:
I renamed this to `Kubernetes 1.27: Query Node Logs Using The Kubelet API` as this is a not a first class Kubernetes API that goes through the API server but is just wired directly through the `proxy/log` endpoint in the kubelet.